package.json

```diff
@@ -83,7 +82,9 @@
     "ethjs-query": "^0.3.8",
     "human-standard-collectible-abi": "^1.0.2",
     "human-standard-token-abi": "^2.0.0",
+    "isomorphic-fetch": "^2.2.1",
     "percentile": "^1.2.1",
+    "uuid": "^3.3.2",
```

Comment:
I couldn't find were are you using this dep. in this PR

Reply:
We were missing it, it's used in TransactionController.